### PR TITLE
fix: Update search bar index and styling

### LINKED ISSUE #17304: Build a wikipedia viewer search leads to 404 NOT FOUND
I can't find the Lesson: _Build a Wikipedia Viewer_, so I used the new search box and I typed "Wikipedia Viewer". The [below ](https://search.freecodecamp.org/?q=wikipedia%20viwer) page open up's with three results. I click on the «Lesson: Build a Wikipedia Viewer» and I get a 404 not found error and the link [here](https://learn.freecodecamp.org/404.html)
Before getting the error above, it looks like the page is loading, with the green freecodecamp bar appearing and disappearing quite fast with white background but with a different url as you can see in the screenshots

screenshot_11
screenshot_2
screenshot_3

#### Tell us about your browser and operating system

* Browser Name:  Firefox
* Browser Version:  60.0.1
* Operating System: Win10

### LINKED ISSUE #34730: Dead link on search results (python map filter reduce)
**Describe your problem and how to reproduce it:**
A link to a specific guide leads to a 404 page

**Add a Link to the page with the problem:**
[Search results page](https://search.freecodecamp.org/?q=reduce%22%0A)
Linked page: Guide: [List map filter reduce](https://guide.freecodecamp.org/python/list-map-filter-reduce)

**Tell us about your browser and operating system:**
* Browser Name: Brave
* Browser Version: Version 0.58.16 Chromium: 71.0.3578.98 (Official Build) (64-bit)
* Operating System: Ubuntu 18.04

### LINKED ISSUE #36606: [.dev] search links to old guide article urls
Screen Shot 2019-08-15 at 2 19 51 PM

Since all the guide articles have been moved to the forum - all the search results for guide articles are linking to 404 pages.

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
Closes #17304
Closes #34730
Closes #36606

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
Just noticed that the search bar is much too tall now. Will update this soon.

**Comment by raisedadead** [comment]
Is this ready?

**Comment by ahmadabdolsaheb** [comment]
@raisedadead I believe so. it is the next in my to do list :)

File: client/src/components/search/searchPage/SearchPageHits.js [hunk 1]
```diff
@@ -9,65 +10,77 @@ import { isEmpty } from 'lodash';
...
+          </a>
+        ))}
       </ul>
     </div>
   ) : (
     <NoResults query={currentRefinement} />
   );
 });
 
+// Pagination kinda sorta working
```
Comment (ahmadabdolsaheb) [inline client/src/components/search/searchPage/SearchPageHits.js]: should we remove the comments?

File: client/src/components/search/searchBar/SearchBar.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 import { createSelector } from 'reselect';
 import { SearchBox } from 'react-instantsearch-dom';
-import { navigate } from 'gatsby';
+// import { navigate } from 'gatsby';
```
Comment (ahmadabdolsaheb) [inline client/src/components/search/searchBar/SearchBar.js]:
```suggestion
```

**Comment by ahmadabdolsaheb** [comment]
Functionality is LGTM. great job @scissorsneedfoodtoo for the integration.

File: client/src/components/search/searchPage/SearchPageHits.js (hunk 1 above)
Comment (scissorsneedfoodtoo) [reply client/src/components/search/searchPage/SearchPageHits.js]: Yes, sorry, let's remove this for now. I can look back at this PR later when we are redoing the Learn search page.

**Comment by raisedadead** [comment]
> 1. When I click on a search result sometimes, there can be a 2-4 second delay before the page even starts to navigate to the news article.  It seems there should be a loading icon to let the user know that something is working.  A few times I thought the page froze.

This should probably be taken up separately as a independent bug. @RandellDawson 
 Can you please create a issue thread for this specific issue? Thanks.

> 2. If type in something and results appear, it might be nice to be able to use the up and down arrow keys to navigate through the results and then press `Enter` to make a selection.

Yes, that would improve the a11y, but again lets take it up as a feature request in a separate issue.

> 3. On the news page is the FCC logo to the right of the search bar and on the other pages, the search bar is to the right of the FCC logo.

This is a work in progress, and the nav will be uniform across all platforms.

**Comment by raisedadead** [comment]
@scissorsneedfoodtoo Does this address any open issues partially or completely?

**Comment by scissorsneedfoodtoo** [comment]
Sorry for the late response. Thank you for your review @RandellDawson. Yes, I was trying to figure out a good way to get keyboard navigation working like it does on Developer News, but wasn't able to work it out at the time. I never ran into the bug where navigating to News takes that long. It's definitely something we should investigate.

Thank you @moT01 for finding those issues. The only other search related issue I was able to find is https://github.com/freeCodeCamp/freeCodeCamp/issues/36606.

**Comment by raisedadead** [comment]
Okay, I have added those issues in the OP for closure.

**Review by raisedadead — APPROVED** [review]

**Pull request merged**